src/app/components/listing/details/compliance/direct-reviews.jsx

```diff
@@ -16,20 +15,22 @@ import { arrayOf } from 'prop-types';
 import { ChplTooltip } from 'components/util';
 import { getDisplayDateFormat } from 'services/date-util';
 import { directReview as directReviewPropType } from 'shared/prop-types';
-import { palette, utilStyles } from 'themes';
+import { palette, utilStyles} from 'themes';
+import theme from 'themes/theme';
```

Comment:
The "theme" object should be able to be imported in the same line as the palette and utilStyles ones